### DISCUSSION TITLE
v0.5.2

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#144] [New Releases]
Changelog:

* Added the option to comment/uncomment lines of code
* Add Chinese translation
* Add search results dock for showing all results in a file
* Reduce Windows distribution package size
* Use default palette for Find Replace Dialog
* Fix crash when restoring floating dock widgets
* Update Scintilla/Lexilla
* Support PREFIX for Linux installation directory

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.5.1...v0.5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.5.2'>v0.5.2</a>.</em>

**Comment by T200proX7** [comment]
Great job. Make a donation link somewhere please. I want to donate and support you making NotepadNext (Notepad++ for Linux). I appreciate and i think many people out there are waiting for it.

**Comment by dail8859** [comment]
@T200proX7 Glad you are finding it useful! Unfortunately I don't have any way to donate at this time, but appreciate the gesture. I guess for now just spread the word :)

**Comment by T200proX7** [comment]
Well, currently it's not really useful yet because a lot is still missing. With PHP the syntax highlighting is too bright. 

Anyway, I am interested in it being fully developed and can offer at least financial support here.

It is also important that the plugins of Notepad++ are supported on this application and the interface is the same. This would give you a complete ecosystem right away.

I've been dreaming of a Notepad++ port that runs native and without Wine on Linux for a long time, and I'm not the only one. At the moment I'm using Geany which doesn't come close to Notepad++.

**Reply by dail8859** [reply]
There is a lot missing but still the application is useful. What the project needs is just more time and more people interested in helping to develop it.

If something like PHP isn't working then feel free to open an issue (I can't fix something I don't know is broken). I don't use PHP at all.

Notepad++ plugin support won't be possible. The two applications are just too different.

**Comment by ZHANGYANG-91** [comment]
I looked for a lot of text compilers to replace notepad++ until I found this one, it's really, really good, I'm going to recommend him to my friends and colleagues, very supportive of the author.
